content/_data/changelogs/weekly.yml

```diff
@@ -14425,6 +14426,77 @@
   # pull: 6271 (PR title: Fixed some deprecation warnings in tests)
   # pull: 6272 (PR title: Deprecate `Util.copyFile()`)
 
+  - version: '2.336'
+    date: 2022-02-21
+    changes:
+      - type: bug
+        category: regression
+        pull: 6289
+        authors:
+          - NotMyFault
+        pr_title: Fix broken script- and console icon in agent view
+        message: |-
+          Restore missing computer icon in the "System Information" view of an agent (regression in 2.335).
+      - type: bug
+        category: regression
+        pull: 6288
+        issue: 67847
+        authors:
+          - benebsiny
+        pr_title: "[JENKINS-67847] booleanRadio button is not shown correctly"
+        message: |-
+          Place text correctly on boolean radio controls (regression in 2.335).
+      - type: bug
+        category: regression
+        pull: 6286
+        issue: 67823
+        authors:
+          - NotMyFault
+        pr_title: "[JENKINS-67823] Don't append leading slash to foreign icons"
+        message: |-
+          Don't add leading slash to foreign icons (regression in 2.335).
+      - type: bug
+        category: regression
+        pull: 6287
+        authors:
+          - NotMyFault
+        pr_title: Don't show status tooltip on nobuilt jobs
+        message: |-
+          Don't show build status on jobs that are not yet built (regression in 2.321).
+      - type: bug
+        category: regression
+        pull: 6283
+        issue: 67837
+        authors:
+          - NotMyFault
+        pr_title: "[JENKINS-67837] Restore labeled agents view"
+        message: |-
+          Use icon of the correct size in list of agents with a specific label (regression in 2.335).
+      - type: bug
+        category: regression
+        pull: 6282
+        issue: 67828
+        authors:
+          - timja
+        pr_title: JENKINS-67828 Don't prepend icon if it's already there
+        message: |-
+          Don't prepend <code>icon</code> when it's already there, resolves missing icons in some cases (regression in 2.335).
```

Comment:
I'm not 1000% happy with this, but not enough to block shipping.  I wonder if it could be 2 sentences or a semicolon after "there".

Reply:
> I'm not 1000% happy with this, but not enough to block shipping. I wonder if it could be 2 sentences or a semicolon after "there".

Good point.  Switched to 2 sentences and regenerated the changelog.